Lib/codeop.py

```diff
@@ -84,9 +84,11 @@ def _maybe_compile(compiler, source, filename, symbol):
     except SyntaxError:
         pass
 
-    # Suppress warnings after the first compile to avoid duplication.
+    # Catch syntax warnings after the first compile
+    # to emit SyntaxWarning at most once.
```

Comment:
Hum, english is not my first language. I would like to say "the warning must be emitted 0 or 1 time, but not more". Is it ok to say "at most once"?

Reply:
I think "at most once" would be correct since "at least once" would mean one or more time.